sys-process/lsof/files/lsof-4.98.0-fix-clang-version-parser.patch

```diff
@@ -0,0 +1,15 @@
+https://bugs.gentoo.org/919253
```

Comment:
Can you send this upstream as a PR please?

Reply:
Done and already merged in https://github.com/lsof-org/lsof/pull/306